test/util.js

```diff
@@ -21,7 +22,9 @@ function peers (nameA, a, nameB, b, async) {
 function bindAll (obj, context) {
   const o = {}
   for (const k in obj) {
-    if (obj[k]) o[k] = obj[k].bind(context)
+    if (allowed.includes(k)) { // simulate which RPC methods will be permitted
+      if (obj[k]) o[k] = obj[k].bind(context)
+    }
```

Comment:
this code is used to help simulate RPC methods. I made sure only those that are permitted are available